playwright/pages/components/InAppGuideDialog.ts

```diff
@@ -0,0 +1,38 @@
+import { Locator, Page } from "@playwright/test";
+
+import { Dialog } from "./Dialog";
+
+export class InAppGuideDialog extends Dialog {
+  override readonly closeButton: Locator;
+
+  constructor(page: Page) {
+    super(page, {
+      title: "Build a page in 5 minutes",
```

Comment:
Should we be hardcoding content like this?

Reply:
What do you mean? It's POM model.

Meaning here you share an instance of the InAppGuideDialog and you define the expected title. If someone change it in the code, test will break because as a user it changed. So it's "normal" but maybe I didn't get your question?